portfolio/src/main/webapp/script.js

```diff
@@ -12,51 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/*
-    The function showNext() allows the button to be clicked twice in order to
-    display a personal summary and a photograph. Because the second click is dependent
-    on the first, a global variable is required to determine when to show the picture.
-    Another roundabout solution would be to check whether the 'summaryCointainer'
-    is empty, eliminating the need for a global varible.
-*/
-var summaryVisble = false;
-var pictureVisbile =false;
+/** 
+ * Allows a button to be clicked successively in order to
+ * display a personal summary, then a photograph. 
+ */
+
 function showNext() {
-    if(summaryVisble == false){
-        showSummary();
-        summaryVisble = true;
-    } 
-    else if(pictureVisbile == false){ 
-        showPicture();
-        pictureVisbile = true;
+  const summaryContainer = document.getElementById('summary');
+  const imageContainer = document.getElementById('picture');
+  if (summaryContainer.innerText.length == 0) {
+    showSummary();
+    } else if (imageContainer.childElementCount == 0) { 
```

Comment:
The `else` block is indented one level too far.

`if`/`else` should be formatted the same as `try`/`catch` in the style guide example: https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks.
See also https://google.github.io/styleguide/jsguide.html#formatting-block-indentation: "When the block ends, the indent returns to the previous indent level."